src/components/filters/filters-from-params.tsx

```diff
@@ -0,0 +1,22 @@
+import { notEmptyString } from '../../value-utils';
+import { Filters, Option } from './filter.types';
+
+export const filtersFromParams = (
+  queryParams: URLSearchParams,
+  availableFilters: Set<string>,
+): Filters => {
+  const filters: Filters = {};
+
+  Array.from(availableFilters).forEach((filterId) => {
+    const value = queryParams.get(filterId);
+    if (value && notEmptyString(value)) {
```

Comment:
The `&& notEmptyString(value)` won't do anything since an empty string is falsey